src/Controller/ExportController.php

```diff
@@ -86,8 +89,22 @@ public function export() : void
                     }
                 }
             }
+            foreach ($val['attributes']['extra'] as $extrafield => $extraval) {
+                if (is_array($extraval)) {
+                    $row[sprintf('extra-%s', $extrafield)] = json_encode($extraval);
```

Comment:
I'd use `extra_` as prefix to have consistent naming
Probaly `extra__` even better to avoid possible conflicts